### ISSUE TITLE
Patternslib release process

### ISSUE THREAD

**Issue by thet** [Patternslib/Patterns#919]
We need a way to automatically update the version number and download reference on Patternslib.com after a new Patternslib release.
It would also be nice to create a ZIP file instead of a tar.gz. file.

/cc @cornae @pilz

**Comment by cornae** [comment]
Yes indeed. Currently I need to clean the packages myself by hand and recompress as a ZIP for each release.
